parser/parser.go

```diff
@@ -88,6 +93,9 @@ func Parse(r io.Reader, pkgName string) (*Report, error) {
 		} else if matches := regexResult.FindStringSubmatch(line); len(matches) == 5 {
 			if matches[4] != "" {
```

Comment:
With the updated regex above we can now check for the string `[build failed]` here and add a failing `Test` to the `tests` slice.